src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,26 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
-     * Imagine that some user wants to login to your site.
+     * Imagine that some user wants to log in to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+
+        if (user != null && user.getPassword() == password) {
+            return true;
+        }
+
         return false;
```

Comment:
Let's remove this redundant 'if' construction and also use equals() method to compare passwords:
```suggestion
        return user != null && user.getPassword.equals(password)
```

Reply:
'if' construction has been changed to a `return user != null && user.getPassword().equals(password)` and also `user.getEmail() == email` in 'if' statement been changed to `user.getEmail().equals(email)` in UserService file.